src/HtmlNode.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace NoTee;
+
+
+interface HtmlNode
+{
+    /**
+     * @return string
+     */
+    public function toString();
```

Comment:
Interface inheritance exists in PHP. But it is not clear to me, how this would make sense here. Could you give me an example?